pkg/graveler/sstable/range_manager.go

```diff
@@ -86,17 +86,20 @@ func (m *RangeManager) GetValueGE(ctx context.Context, ns committed.Namespace, i
 	defer m.execAndLog(ctx, it.Close, "close iterator")
 
 	// Ranges are keyed by MaxKey, seek to the range that might contain key.
-	key, value := it.SeekGE(lookup)
+	key, value := it.SeekGE(lookup, sstable.SeekGEFlags(0))
```

Comment:
Here and elsewhere
```suggestion
	key, value := it.SeekGE(lookup, sstable.SeekGEFlagsNone)
```

Reply:
Can't access `SeekGEFlagsNone` it is part of the internal package.
A lot of the sstable implementation is inside `/internal` which we don't have access - unless an alias or equal value was re-defined under `sstable` package.